templates/submission/submissionMetadataFormTitleFields.tpl

```diff
@@ -21,6 +29,6 @@
 	{/fbvFormSection}
 	{fbvFormSection description="common.subtitle.tip"}{/fbvFormSection}
 	{fbvFormSection title="common.abstract" for="abstract" required=$abstractsRequired}
-		{fbvElement type="textarea" multilingual=true name="abstract" id="abstract" value=$abstract rich="extended" readonly=$readOnly}
+		{fbvElement type="textarea" multilingual=true name="abstract" id="abstract" value=$abstract rich=$rich readonly=$readOnly}
```

Comment:
I see. That'll be a problem if there is rich content, because turning off `rich` will result in HTML tags etc. being escaped and displayed to the reader literally.

I can think of two approaches, both best solved in the form builder vocab code:
- Configure TinyMCE to behave read-only as expected when `readonly` is set, or
- have the form builder vocab present a `<div>` element or something, with the contents of the field inside it, and not use TinyMCE at all for cases where `readonly` is set